cmd/thanos/query.go

```diff
@@ -83,6 +83,9 @@ func registerQuery(m map[string]setupFunc, app *kingpin.Application, name string
 	enableAutodownsampling := cmd.Flag("query.auto-downsampling", "Enable automatic adjustment (step / 5) to what source of data should be used in store gateways if no max_source_resolution param is specified. ").
 		Default("false").Bool()
 
+	enablePartialResponse := cmd.Flag("query.partial-response", "Enable partial response for queries if no partial_response is specified.").
```

Comment:
Could be worded like  ̀Enable partial response for queries if no partial_response param is specified.` to mimic above ?